### ISSUE TITLE
[Suggestion] Acceleration Changes

### ISSUE THREAD

**Issue by ZachZent** [tsunamayo/Starship-EVO#1972]
Ship accelerate is a bit backwards right now. You take off from standing still at an insane rate while getting those last few meters per second takes way too long. The red graph below shows what ship acceleration currently is.

![](https://cdn.discordapp.com/attachments/406664114249793536/707132109009059910/unknown.png)

The green graph is what we should have. A slower start before picking up speed and less time before reaching max speed.

This should also depend on the mass of the ship where a fighter can get up to speed faster, but not as fast as it is currently) compared to a larger ship.

**Comment by Battlepixel** [comment]
I agree with you that the acceleration curve flattens very much towards the rear. But for the boost mode, an initial small increase would be very bad. In order to cover a reasonable distance in the PVP before the next hit run, the initial boost from low speeds should also bring about the greatest acceleration. I agree with you, however, that for normal (non-boost mode) flight mode, a little less acceleration at low speeds would be good for precise maneuvering(
albeit not important to me at the moment).

**Comment by tsunamayo** [comment]
Yes you are correct - I wanted to update it to the first curve. Pb is I can maneuver just fine with my lil shuttle, but with dwarf shuttle for example it is indeed too sensitive. Also I could use a different curve for the boost mode.

**Comment by YnotnA** [comment]
I do not know how the displacement is managed but would it not be possible to carry out a "push" according to the power of the reactors? Remember when I was studying we had calculations to manage this kind of case : http://villemin.gerard.free.fr/aScience/Physique/AccelPhy.htm#formule
(sorry it's in french but I don't think it will be a problem for you;))

**Comment by ProPeach** [comment]
I think a different curve for boost and maneuver mode is a good idea, boost mode can keep the curve of the red graph while maneuver mode should be a little more gradual
